profile.md

```diff
@@ -54,6 +59,75 @@ the [Open Service Broker API specification](spec.md) SHALL take precedence.
 
 - *Application*: The software that uses a Service Instance via Service Binding.
 
+## Originating Identity Header
+
+In the [Open Service Broker API specification](spec.md) it defines an
+additional HTTP Header that can be included in messages from the platform
+to identify the user that requested the action to be taken.
+
+The header consists of two parts: a `platform` string and `value` string.
+Both values will vary based on the platform which is being used. The
+following sections define the values to be used based on the Platform
+and which properties are expected to appear in the JSON encoded
+`value`.
+
+Note that when both the originating identity HTTP Header and the Context
+object appear in the same message the `platform` value MUST be the same
+for both.
+
+### Cloud Foundry
+
+*`platform` Value*: `cloudfoundry`
+
+The following properties MUST appear within the JSON encoded `value`:
```

Comment:
I read both `user_name` and `user_id` MUST be provided by CF platforms to be compliant with the cloudfoundry profile.

 I can see cases where a cloudfoundry user would not want his user_name to be disclosed to untrusted brokers: e.g. `user_name` can be used to trigger denial of service by locking account after multiple unsuccessful login form attempts that use this `user_name` as user id. 

More specifically, a broker might be untrusted when offered by a 3rd party public Saas service provider, (provisionned by CF admins or possibly by space developpers through space private brokers), while some other corporate trusted brokers might indeed legitimaly need `user_name`.

Was it considered to publish to platform users the broker's intent to leverage `user_name`, e.g. by leveraging the existing catalog `requires ` key ?

Was it considered that the OSB spec provides rooms for cloudfoundry to omit `user_name` while remaining compliant ? For instance, during broker registration, admin may choose to opt-opt from exposing user_name to this broker.

Reply:
done